charm/trusty/calico-docker/playbooks/config-changed.yaml

```diff
@@ -0,0 +1,41 @@
+---
+
+    - set_fact:
+        db_relation: "{{ relations.db | default([]) }}"
+
+    - set_fact:
+        etcd_authority: "{{ relations.db[0].hostname }}:{{ relations.db[0].port }}"
+      when: db_relation != []
+
+
+    - name: Pull CalicoNode image
+      docker:
+          name=caliconode
+          image={{ docker_image }}
+          state=present
+
+    - name: Pull CalicoCTL
+      get_url: url={{calicoctl_package}} sha256sum={{calicoctl_sum}} dest="/usr/local/bin/calicoctl"
```

Comment:
For the sake of security we're *probably* ok because we're using the sha256 sum, but we're still technically a bit vulnerable. For now, that'll have to do.